src/syzygy/syzygy.cc

```diff
@@ -421,7 +421,7 @@ void init_indices() {
   }
   // TODO: choose a good seed.
   std::mt19937 gen(123523465);
-  std::uniform_int_distribution<Key> dist(std::numeric_limits<Key>::lowest(),
+  const std::uniform_int_distribution<Key> dist(std::numeric_limits<Key>::lowest(),
```

Comment:
g++/clang on linux complain about this (and the other `const` on distributions in `random.cc`)

Reply:
I'll look into it.

Reply:
I tried a variety of way to repro this in windows, including using the intel compiler, and attempting to set up a Linux sub system, but to no avail. So, I'm simply fixing the issues as demonstrated in the build log.

Reply:
You missed the one in `syzygy.cc`.